lib/active_admin/error.rb

```diff
@@ -19,6 +19,16 @@ def message
     end
   end
 
+  class ScopeNotDefined < StandardError
```

Comment:
I'd like to include "Policy" in the error class name so it's clear since when I think "scopes" it's something else. Perhaps "PolicyScopeNotDefined" as the class name?

Reply:
I understand your concern with the name. I'm open to use another one. I just want to avoid `Policy` because it's a  `Pundit` strong word. I want to model something more abstract, the `ActiveRecord::RecordNotFoundError` for `scope_collection` method. I don't want to tailor the error for a specific AuthorizationAdapter implementation.

I renamed the class to `ScopeAuthorizationError`. Is it better?